db/schema.rb

```diff
@@ -10,13 +10,14 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_07_02_125917) do
+ActiveRecord::Schema.define(version: 2020_07_06_172428) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "citext"
   enable_extension "fuzzystrmatch"
   enable_extension "pgcrypto"
   enable_extension "plpgsql"
+  enable_extension "postgis"
```

Comment:
Not sure how/why this and the table below have crept in?

Reply:
Good catch, it was stuff that entered my local database after I put a production dump into it. But I don't know why the production database has this stuff in it. @cesidio do you know why?

Reply:
Not sure why, as the extensions we ask for are only these
`"enable_extensions": ["pgcrypto", "fuzzystrmatch", "plpgsql"]`
maybe it's a default on PaaS